Index: test.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"hello_world\")\r\nprint(\"hello_world\")
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/test.py b/test.py
--- a/test.py	(revision aec47348cb31aff2d8fcf4f62a59afeab0c00240)
+++ b/test.py	(date 1678397100979)
@@ -1,2 +1,3 @@
 print("hello_world")
-print("hello_world")
\ No newline at end of file
+print("hello_world")
+NHNNHN
\ No newline at end of file
